diff --git a/ckan/controllers/group.py b/ckan/controllers/group.py
index af36717..07a5afd 100644
--- a/ckan/controllers/group.py
+++ b/ckan/controllers/group.py
@@ -204,10 +204,11 @@ class GroupController(base.BaseController):
 
         q = c.q = request.params.get('q', '')
         # Search within group
+        owner_fq = ''
         if c.group_dict.get('is_organization'):
-            q += ' owner_org:"%s"' % c.group_dict.get('id')
+            owner_fq += ' owner_org:"%s"' % c.group_dict.get('id')
         else:
-            q += ' groups:"%s"' % c.group_dict.get('name')
+            owner_fq += ' groups:"%s"' % c.group_dict.get('name')
 
         c.description_formatted = h.render_markdown(c.group_dict.get('description'))
 
@@ -279,7 +280,7 @@ class GroupController(base.BaseController):
             if (c.group and c.group.id in user_member_of_orgs):
                 fq = ''
                 context['ignore_capacity_check'] = True
-
+            fq += owner_fq
             facets = OrderedDict()
 
             default_facet_titles = {'organization': _('Organizations'),
